controllers/user/utility.go

```diff
@@ -53,22 +53,18 @@ func GetUser(userId string) (*User, error) {
 	return &user, nil
 }
 
-func GetUsersWithRole(roleName string) (*[]User, error) {
+func GetUsersWithRole(roleId string) (*[]User, error) {
 	cfg, err := config.Get()
```

Comment:
hoping this doesnt break anythign in my code!

Reply:
ahahahaha....